tests/basic/storage_pool_test.go

```diff
@@ -403,18 +395,11 @@ func poolResizeIsInProgress(poolToBeResized *api.StoragePool) bool {
 			dash.VerifyFatal(len(pools) > 0, true, "Validate storage pools exist")
 
 			updatedPoolToBeResized := pools[poolToBeResized.Uuid]
-
 			dash.VerifyFatal(updatedPoolToBeResized != nil, true, "Validate pool to be resized exist")
-			dash.VerifyFatal(waitCount, 0, "timed out waiting for pool resize to finish")
 			if updatedPoolToBeResized.LastOperation.Status != api.SdkStoragePool_OPERATION_SUCCESSFUL {
 				dash.VerifyFatal(updatedPoolToBeResized.LastOperation.Status != api.SdkStoragePool_OPERATION_FAILED, true, fmt.Sprintf("PoolResize has failed. Error: %s", updatedPoolToBeResized.LastOperation))
-				err = ValidatePoolRebalance()
-				if err != nil {
-					dash.VerifyFatal(err, nil, "Error while pool rebalance")
-				}
```

Comment:
looks like something wrong here, as you remove some steps from the test, rebase issue?